string cam_id
int32 fps
---
